Lib/test/test_ast.py

```diff
@@ -565,6 +565,12 @@ def test_no_fields(self):
         x = ast.Sub()
         self.assertEqual(x._fields, ())
 
+    # TODO: RUSTPYTHON
+    # In CPython classes in _ast with the exception of `AST` are created dynamically
+    # (by calling PyType_Type, see line 790 in asdl_c.py). As a result, they have the
+    # HEAPTYPE flag set and copyreg pickles them. We currently create them without the
+    # flag, apparently.
```

Comment:
This part looks more like discussion for the issue. Could you open an issue about this and start the thread with this comment? Then other contributors who find something about this can post their opnions easier than comments.

Reply:
Yup. I'll try and open an issue for this at some point.